diff -ur drivel-1.3.90-orig/data/gnome-drivel.desktop.in drivel-1.3.90/data/gnome-drivel.desktop.in
--- drivel-1.3.90-orig/data/gnome-drivel.desktop.in	2005-01-02 23:29:29.000000000 -0700
+++ drivel-1.3.90/data/gnome-drivel.desktop.in	2005-05-29 07:55:47.000000000 -0600
@@ -4,11 +4,11 @@
 _GenericName=Journal Editor
 _Comment=Update and manage your online journal
 Exec=drivel %U
-Terminal=0
+Terminal=false
 Type=Application
 Icon=livejournal.png
 MimeType=application/x-drivel
-Categories=GNOME;Application;Network;
+Categories=GTK;GNOME;Network;
 StartupNotify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=drivel
Only in drivel-1.3.90/data: gnome-drivel.desktop.in.orig
Only in drivel-1.3.90/data: gnome-drivel.desktop.in.rej
