Index: src/network.c
===================================================================
RCS file: /cvs/gnome/drivel/src/network.c,v
retrieving revision 1.50
diff -p -u -r1.50 network.c
--- src/network.c	30 May 2005 03:25:35 -0000	1.50
+++ src/network.c	1 Jun 2005 00:20:42 -0000
@@ -219,7 +219,23 @@ setup_proxies (DrivelClient *dc, CURL *s
 	else
 		pass = NULL;
 	userpwd = NULL;
-	url = g_strdup (dc->proxy_url);
+	if (dc->proxy_url)
+	{
+		gchar *colon1, *colon2;
+		
+		/* Check for an IPv6 URL by searching for at least two colons (:) */
+		colon1 = strstr (dc->proxy_url, ":");
+		if (colon1)
+			colon2 = strstr (colon1 + 1, ":");
+		else
+			colon2 = NULL;
+		if (colon2)
+			url = g_strdup_printf ("[%s]", dc->proxy_url);
+		else
+			url = g_strdup (dc->proxy_url);
+	}
+	else
+		url = NULL;
 	port = dc->proxy_port;
 	use_auth = dc->proxy_auth;
 	g_mutex_unlock (net_mutex);
